k8sutil/test_utils.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class FakeException(Exception):
+    pass
+
+
+class FakeClass(object):
```

Comment:
This looks an awful lot like the start of a mocking library.  Any reason not to use an existing one?  Or better yet, skip mocking and focus on writing code that can be unit tested without mocks?  It would be simpler to write and easier to maintain.